### PR TITLE
[KOGITO-2437] - Migarate namespace-cleaning job to rhba jenkins

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-2437

Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

### CODE REVIEW

**Comment by Kaitou786** [comment]
@msmagnanijr shouldn't we also make changes in the script or in the pipeline to authenticate with openshift.?

**Comment by radtriste** [comment]
yes we should

**Comment by radtriste** [comment]
from my PRs to migrate jobs, I am using some credentials:
https://github.com/kiegroup/kogito-images/pull/182/files#diff-4c6c57cd8094d1a0078f1cfa0e02fcf9R38

- OPENSHIFT_API
- OPENSHIFT_CREDS

I think we should sync on that

**Comment by msmagnanijr** [comment]
> from my PRs to migrate jobs, I am using some credentials:
> https://github.com/kiegroup/kogito-images/pull/182/files#diff-4c6c57cd8094d1a0078f1cfa0e02fcf9R38
> 
> * OPENSHIFT_API
> * OPENSHIFT_CREDS
> 
> I think we should sync on that

Okay! I'll take a look :)

**Comment by msmagnanijr** [comment]
> @msmagnanijr shouldn't we also make changes in the script or in the pipeline to authenticate with openshift.?

yep! It was that before we had that node that was already authenticated but yes, it makes sense now to have authentication.

File: clean-old-namespaces-Jenkinsfile [hunk 1]
```diff
@@ -1,13 +1,24 @@
 pipeline{
-    agent {label 'operator-slave'}
+    agent { label 'kogito-operator-slave && !master'}
```
Comment (Kaitou786) [inline clean-old-namespaces-Jenkinsfile]: Can we use `kogito-cloud` label over here?

**Review by Kaitou786 — CHANGES_REQUESTED** [review]

**Comment by Kaitou786** [comment]
Apart from label LGTM

**Review by Kaitou786 — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**